quickcheck-dynamic-iosim/quickcheck-dynamic-iosim.cabal

```diff
@@ -41,6 +41,20 @@ common lang
     ImportQualifiedPost
     ScopedTypeVariables
     StandaloneDeriving
+    DerivingStrategies
+    FlexibleContexts
+    FlexibleInstances
+    GADTs
+    KindSignatures
+    MultiParamTypeClasses
+    TupleSections
+    TypeFamilies
+    TemplateHaskell
```

Comment:
Do we really use this?

Reply:
That's a good question - I just moved things out of files and removed the most "wtf" ones... I'll have a look at which ones are really necessary and clean the lists up a bit.